### ISSUE TITLE
PropertyNamingPolicy null walue

### ISSUE THREAD

**Issue by amoraller** [tochka-public/Tochka.JsonRpc#93]
I think, we should support unsetted value for `JsonRpcServerOptions.DefaultDataJsonSerializerOptions.PropertyNamingPolicy `

Today,  i use this policy

```csharp
class UnsetNamingPolicy : JsonNamingPolicy
{
    public override string ConvertName(string name) => name;
}
```

For example System.Text.Json support null value for policy
https://github.com/dotnet/corefx/blob/e667c29636a622eb4f9493f75232b44e0ae90b29/src/System.Text.Json/src/System/Text/Json/Serialization/JsonPropertyInfo.cs#L106-L119

**Comment by OptimumDev** [comment]
Hello! Thank you for the issue.
You are right, null PropertyNamingPolicy should be supported and we tried to achive it, but looks like we missed couple of spots.
We will fix it when we will able to, or you can contribute by PR 😉

**Comment by OptimumDev** [comment]
Sorry for waiting so long, it's now fixed in version [6.5.1](https://github.com/tochka-public/Tochka.JsonRpc/releases/tag/v6.5.1)
